Kiosk/App/AppDelegate+GlobalActions.swift

```diff
@@ -6,23 +6,37 @@ func appDelegate() -> AppDelegate {
     return UIApplication.sharedApplication().delegate as AppDelegate
 }
 
+typealias AnimationCompletion = () -> ()
```

Comment:
Wanted to make an extension on this, or even one on `Optional<AnimationCompletion>`, but noooo Swift won't let me. Can't wait for 1.2.